### ISSUE TITLE
bestcontentsite.top

### ISSUE THREAD

**Issue by FadeMind** [hagezi/dns-blocklists#3205]
### Which domain(s) should be blocked?

bestcontentsite.top

### Why should these domain(s) be blocked?

Full URL: ` https://a.bestcontentsite.top/loader?a=1275305&s=1307954&t=101&p=6083 `
Blocked by Dandelion Sprout's Anti-Malware List (and TLDs from you, but Adguard DNS record only Dandelion lists in query log)

TLD rules from `https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Alternate%20versions%20Anti-Malware%20List/AntiMalwareAdGuardHome.txt`
```
||*.top^$denyallow=caitlin.top|callmebymygender.top|corriente.top|gdtot.top|nicenature.top|reminder.top|magocoro.top|castlevania.top|suiten.top|shucks.top|1stream.top|ambr.top|techblog.top|changlam10.top|changlam11.top|pdcdn1.top|mastodon.top|pressplay.top|chillx.top|strims.top|thedesk.top|audioforyou.top|pegelinux.top|awavenue.top|reyhub.top
```
![20240718-1721322017](https://github.com/user-attachments/assets/87bf422c-8f58-4ddc-819f-383e5a961a01)

In your @hagezi TLDs spam ( `https://raw.githubusercontent.com/hagezi/dns-blocklists/main/adblock/spam-tlds.txt `) The number of exclusions is lower.
```
||*.top^$denyallow=1stream.top|ambr.top|caitlin.top|chillx.top|corriente.top|mastodon.top|nicenature.top|pegelinux.top|pressplay.top|reminder.top|shucks.top|strims.top|suiten.top|thedesk.top|vsdb.top
```

### Privacy

- [X] I confirm that the report does not contain any private information.

**Comment by hagezi** [comment]
AdGuard DNS only shows the list where the first match was found. This does not mean that the domain is not also on other lists. Currently blocked via Ultimate and TLD list. I will make improvements in this regard.

As for the exclusions in the TLD list, I have cleaned up dead, parked and unpopular domains to keep the denyallow modifiers as short as possible, as the whole rule becomes invalid above a certain length.
